Classes/ScrnBalance.uc

```diff
@@ -3205,51 +3034,83 @@ function ServerTraveling(string URL, bool bItems)
     if (NextMutator != None)
         NextMutator.ServerTraveling(URL,bItems);
 
-    if ( ScrnGT == none || ScrnGT.ScrnGameLength == none )
+    Cleanup();
+}
+
+simulated function Destroyed()
+{
+    log("ScrnBalance destroyed", 'ScrnBalance');
+    super.Destroyed();
+}
+
+
+final protected simulated function Cleanup()
+{
+    local int j;
+
+    ScrnGT = none;
```

Comment:
This breaks `if (ScrnGT == none` statement a few lines below. No need to reset those links - let them live until the Garbage Collector kicks in.